config/artifact-caching-proxy_aws.yaml

```
@@ -0,0 +1,58 @@
+ingress:
+  hosts:
+    - host: repo.aws.jenkins.io
+      paths:
+        - path: /
+          pathType: Prefix
+  tls:
+    - secretName: tls-secret
+      hosts:
+        - repo.aws.jenkins.io
+
+persistence:
+  enabled: true
+  size: 100
+  storageClass: gp2
```

```suggestion
  storageClass: gp3
```

IF we use an EBS volume (which requires the EBS controller installed and set up with OIDC)